generic/Emscripten-wasm.cmake

```diff
@@ -9,6 +9,10 @@
 #
 
 set(CMAKE_SYSTEM_NAME Emscripten)
+set(CMAKE_SYSTEM_VERSION 1)
+
+set(CMAKE_CROSSCOMPILING TRUE)
+set_property(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS FALSE)
```

Comment:
This is not true. You can have shared libs on Emscripten.

Reply:
this was extracted from the official emscripten platform toolchain